cypress/e2e/permit2.test.ts

```diff
@@ -81,6 +81,31 @@ describe('Permit2', () => {
     })
   })
 
+  it('retains modal state when the window loses focus', () => {
```

Comment:
Any chance you could rebase this over https://github.com/Uniswap/interface/pull/6656 to avoid the conflict?